packages/shylu/shylu_node/tacho/src/impl/Tacho_SupernodeInfo.hpp

```diff
@@ -335,7 +335,7 @@ template <typename ValueType, typename DeviceType> struct SupernodeInfo {
               {
                 const ordinal_type lbeg = ap(i), lend = ap(i + 1);
                 ordinal_type *first = &aj(lbeg);
```

Comment:
I guess the old code caused Kokkos bounds check errors? A cleaner way to get offset pointers without dereferencing might be:
```
ordinal_type *first = aj.data() + lbeg;
ordinal_type *last = aj.data() + lend;
```